modules/guides/pages/cbo.adoc

```diff
@@ -0,0 +1,208 @@
+= Cost-Based Optimizer
+:page-topic-type: guide
+:page-partial:
+:page-pagination: prev
+:imagesdir: ../assets/images
+:page-edition: Enterprise Edition
+:description: How to use the Cost-Based Optimizer and manage optimizer statistics.
+
+[abstract]
+{description}
+
+== Introduction
+
+// tag::intro[]
+In Couchbase Server Enterprise Edition, the Cost-Based Optimizer enables the Query service to create the most efficient plan to execute a query.
+// end::intro[]
+
+The Cost-Based Optimizer can generate a query plan for
+xref:n1ql-language-reference/selectintro.adoc[SELECT],
```

Comment:
I tested these out in the page preview and the refs/links don't work. 
From what I can see [here](https://github.com/couchbase/docs-server/tree/release/7.1/modules/n1ql/pages/n1ql-language-reference), this should be:

```suggestion
xref:n1ql:n1ql-language-reference/selectintro.adoc[SELECT],
```
Same for the others below.

Reply:
Done